docs/pages/components/form.md

```diff
@@ -851,3 +851,133 @@ Do not use the checkbox control if:
 {% include display-component.html component=checkbox %}
 
 <br>
+
+## Form with 4-7-1 Layout
+
+{% capture form-edit %}
+<div class="fd-container fd-form-layout-grid-container">
+    <div class="fd-row">
+        <div class="fd-col fd-col--4 fd-form-layout-grid-flex fd-form-layout-grid-flex--right">
+          <label class="fd-form-label" for="input-1">Default Input:</label>
+        </div>
+        <div class="fd-col fd-col--7">
+            <input class="fd-input" type="text" id="input-1" placeholder="Field placeholder text">
+        </div>
+    </div>
+    <div class="fd-row">
+        <div class="fd-col fd-col--4 fd-form-layout-grid-flex fd-form-layout-grid-flex--right">
+          <label class="fd-form-label fd-form-label--required" for="input-1c">Required Input: </label>
+        </div>
+        <div class="fd-col fd-col--7">
+            <input class="fd-input" type="text" id="input-1" placeholder="Field placeholder text">
+        </div>
+    </div>
+    <div class="fd-row">
+        <div class="fd-col fd-col--4 fd-form-layout-grid-flex fd-form-layout-grid-flex--right">
+          <label class="fd-form-label" for="input-2">2 Inputs: </label>
+        </div>
+        <div class="fd-col fd-col--7">
+            <div class="fd-row">
+                  <div class="fd-col fd-col--6">
+                      <input class="fd-input" type="text" id="input-2" placeholder="Field placeholder text">
+                  </div>
+                  <div class="fd-col fd-col--6">
+                      <input class="fd-input" type="text" id="input-3" placeholder="Field placeholder text">
```

Comment:
I'm not sure if this is just to demonstrate what 2 inputs look like in the layout, but `input-3` would need a label.

Reply:
fixed